TombEngine/Scripting/Internal/TEN/Objects/Lara/LaraObject.cpp

```diff
@@ -85,8 +85,8 @@ void LaraObject::SetWet(sol::optional<int> wetness)
 {
 	auto* lara = GetLaraInfo(m_item);
 
-	unsigned char value = wetness.has_value() ? (unsigned char)wetness.value() : UCHAR_MAX;
-	for (unsigned char& i : lara->Wet)
+	float value = wetness.has_value() ? (float)wetness.value() : PLAYER_DRIP_NODE_MAX;
```

Comment:
shall I test this?

Reply:
Yes please. The Lua function should also be modified to work with floats.